clouds/aws/utilities/rosa_cluster_driver.py

```diff
@@ -0,0 +1,68 @@
+import os
+from datetime import datetime, timedelta
+
+from rosa.cli import execute
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+
+def build_command_with_params(
+    cluster_name,
+    cluster_expiration,
+    replicas,
+    compute_machine_type,
+    openshift_version,
+    aws_region,
+    channel_group,
+):
+    return (
```

Comment:
`rosa create cluster` has many more options than these
please add support to all